src/BeadingStrategy/OuterWallInsetBeadingStrategy.cpp

```diff
@@ -1,16 +1,13 @@
-//Copyright (c) 2022 Ultimaker B.V.
-//CuraEngine is released under the terms of the AGPLv3 or higher.
+// Copyright (c) 2023 UltiMaker
+// CuraEngine is released under the terms of the AGPLv3 or higher
 
 #include "BeadingStrategy/OuterWallInsetBeadingStrategy.h"
 
 #include <algorithm>
 
 namespace cura
 {
-OuterWallInsetBeadingStrategy::OuterWallInsetBeadingStrategy(coord_t outer_wall_offset, BeadingStrategyPtr parent) :
-    BeadingStrategy(*parent),
-    parent(std::move(parent)),
-    outer_wall_offset(outer_wall_offset)
+OuterWallInsetBeadingStrategy::OuterWallInsetBeadingStrategy(coord_t outer_wall_offset, BeadingStrategyPtr parent) : BeadingStrategy(*parent), parent(std::move(parent)), outer_wall_offset(outer_wall_offset)
```

Comment:
So always do default constructor stuff on newlines. Try to figure out if we can do an exception if there is only one